backend/src/types.ts

```diff
@@ -624,6 +624,14 @@ export type ImageTagInfo = {
 
 export type ImageType = 'byon' | 'jupyter' | 'other';
 
+export type StorageClassConfig = {
```

Comment:
Doesn't this already exist in `src/k8sTypes`?

Reply:
this is for the backend. they dont share the same type files